apps/rahat/src/projects/project.service.ts

```diff
@@ -105,6 +106,16 @@ export class ProjectService {
           { cmd: BeneficiaryJobs.LIST_BY_PROJECT },
           { projectId: uuid, ...payload }
         );
+
```

Comment:
@avash-rumsan  There has been some refactor by removing switch cases on handling project actions. please resolve the conflict